src/engine/SCons/Tool/MSCommon/common.py

```diff
@@ -173,8 +172,14 @@ def normalize_env(env, keys, force=False):
     if sys32_wbem_dir not in normenv['PATH']:
         normenv['PATH'] = normenv['PATH'] + os.pathsep + sys32_wbem_dir
 
-    debug("PATH: %s" % normenv['PATH'])
+    # Without Powershell in PATH, an internal call to a telemetry
+    # function (starting with a VS2019 update) can fail
+    # Note can also set VSCMD_SKIP_SENDTELEMETRY to avoid this.
+    sys32_ps_dir = os.path.join(sys32_dir, r'WindowsPowerShell\1.0')
```

Comment:
this should be WindowsPowerShell\v1.0

Reply:
Sigh. Yes, I missed it, despite looking right at it.